lib/web_ui/skwasm/text/paragraph.cpp

```diff
@@ -58,6 +61,46 @@ SKWASM_EXPORT int32_t paragraph_getPositionForOffset(Paragraph* paragraph,
   return position.position;
 }
 
+SKWASM_EXPORT bool paragraph_getClosestGlyphInfoAtCoordinate(
+    Paragraph* paragraph,
+    SkScalar offsetX,
+    SkScalar offsetY,
+    SkScalar* graphemeLayoutBounds,
+    size_t* garphemeCodeUnitRange,
```

Comment:
There is a typo in this name

Also, could you just put a comment next to the argument that explains it is an array of length two that represents the start and end.